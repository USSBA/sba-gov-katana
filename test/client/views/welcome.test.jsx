/*global expect*/

/*eslint-disable no-unused-vars*/
import React from 'react';
/*eslint-enable no-unused-vars*/
import Welcome from '../../../client/views/welcome.jsx';
import renderer from 'react-test-renderer';

console.log(Welcome);

test('Welcome displays a name', () => {
  const component = renderer.create(
    <Welcome name="Jon" />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();

});