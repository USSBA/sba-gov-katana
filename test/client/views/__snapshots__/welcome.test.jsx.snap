exports[`test Welcome displays a name 1`] = `
<h1
  className="app">
  Hello, 
  Jon
</h1>
`;
